packages/driver/src/multi-domain/events/logs.ts

```diff
@@ -2,20 +2,10 @@ import { LogUtils } from '../../cypress/log'
 
 export const handleLogs = (Cypress: Cypress.Cypress) => {
   const onLogAdded = (attrs) => {
-    // TODO:
-    // - handle printing console props (need to add to runner)
-    //     this.runner.addLog(args[0], this.config('isInteractive'))
-
     Cypress.specBridgeCommunicator.toPrimary('log:added', LogUtils.getDisplayProps(attrs))
   }
 
   const onLogChanged = (attrs) => {
-    // TODO:
-    // - add invocation stack if error:
-    //     let parsedError = correctStackForCrossOriginError(log.get('err'), this.userInvocationStack)
-    // - notify runner? maybe not
-    //     this.runner.addLog(args[0], this.config('isInteractive'))
-
```

Comment:
This should all be covered in [this PR](https://github.com/cypress-io/cypress/pull/20949).